<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_IP">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Direccion IP         :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_masc">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mascara de subred :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_IP_primer">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_IP_segun">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_IP_tercer">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_IP_cuarto">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_MC_primer">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_MC_segun">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_MC_cuarto">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_MC_tercer">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_MC_bits">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ln_interludio">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QTableView" name="tb_capturar_calculadora">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>451</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_mostrar">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostrar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
